components/google_drive/actions/create-file/create-file.js

```diff
@@ -0,0 +1,248 @@
+      default: false,
+    },
+    ocrLanguage: {
+      type: "string",
+      label: "OCR Language",
+      description:
+        "A language hint for OCR processing during image import (ISO 639-1 code).",
+      optional: true,
+    },
+    useContentAsIndexableText: {
+      type: "boolean",
+      label: "Use Content As Indexable Text",
+      description:
+        "Whether to use the uploaded content as indexable text.",
+      default: false,
+    },
+    supportsAllDrives: {
+      type: "boolean",
+      label: "Supports All Drives",
+      description:
+        "Whether to include shared drives. Set to 'true' if saving to a shared drive. Defaults to 'false' if left blank.",
+      optional: true,
+    },
+    contentHintsIndexableText: {
+      type: "string",
+      label: "Content Hints Indexable Text",
+      description:
+        "Text to be indexed for the file to improve fullText queries. This is limited to 128KB in length and may contain HTML elements.",
+      optional: true,
+    },
+    contentRestrictionsReadOnly: {
+      type: "boolean",
+      label: "Content Restrictions Read Only",
+      description:
+        "Whether the content of the file is read-only. If a file is read-only, a new revision of the file may not be added, comments may not be added or modified, and the title of the file may not be modified.",
+      optional: true,
+    },
+    contentRestrictionsReason: {
+      type: "string",
+      label: "Content Restrictions Reason",
+      description:
+        "Reason for why the content of the file is restricted. This is only mutable on requests that also set readOnly=true.",
+      optional: true,
+    },
+    copyRequiresWriterPermission: {
+      type: "boolean",
+      label: "Copy Requires Writer Permission",
+      description:
+        "Whether the options to copy, print, or download this file, should be disabled for readers and commenters.",
+      optional: true,
+    },
+    description: {
+      type: "string",
+      label: "Description",
+      description: "A short description of the file.",
+      optional: true,
+    },
+    folderColorRgb: {
+      type: "string",
+      label: "Folder Color RGB",
+      description:
+        "The color for a folder as an RGB hex string. If an unsupported color is specified, the closest color in the palette will be used instead.",
+      optional: true,
+    },
+    mimeType: {
+      type: "string",
+      label: "Mime Type",
+      description: `The MIME type of the file. Google Drive will attempt to automatically detect an appropriate value from uploaded content if no value is provided. The value cannot be changed unless a new revision is uploaded. If a file is created with a Google Doc MIME type, the uploaded content will be imported if possible. 
```

Comment:
Need options array for this prop

Reply:
Created file google-mime-types.js for Google specific mime types and used require("mime-db") for the rest. It's too much data to load all at once, so I have it loading 10 per "page".